packages/rainbowkit/src/components/ConnectOptions/MobileOptions.tsx

```diff
@@ -150,6 +212,7 @@ function WalletButton({
 enum MobileWalletStep {
   Connect = 'CONNECT',
   Get = 'GET',
+  WalletConnect = 'WALLETCONNECT',
 }
```

Comment:
We shouldn't need to modify this control tree if we route to this component at the top-level in `ConnectOptions`

Reply:
Good point 🙏